modules/local_storage/src/StockLocationStorage.php

```diff
@@ -34,4 +34,48 @@ public function loadEnabled(PurchasableEntityInterface $entity) {
     return $enabled_locations;
   }
 
+  public function loadInContext(Context $context, PurchasableEntityInterface $entity) {
+
+    $store = $context->getStore();
+    // Make sure we have the availability field for the location.
+    if ($store->hasField('field_available_stock_locations')) {
```

Comment:
To support multiple stores you must add **field_available_stock_locations** to your store